pkg/controllermanager/controller/bastion/reconciler.go

```diff
@@ -112,6 +112,10 @@ func (r *Reconciler) Reconcile(ctx context.Context, request reconcile.Request) (
 		}
 	}
 
+	if requeueAfter < 0 {
+		requeueAfter = 0
```

Comment:
Hm, don't we want to requeue? Then it must be >0.

Reply:
True.